docs/source/noise/row_noise.rst

```diff
@@ -7,13 +7,66 @@ Row-based Noise
 Row-based noise operates on one row of data at a time, for example by omitting
 or duplicating entire rows.
 
+Types of row-based noise:
+
+.. contents::
+   :local:
+
+Do not respond
+--------------
+
+Sometimes people don't respond to a census or survey questionnaire and are
+unable to be reached by other means such as telephone or personal visit. For the
+Decennial Census and household surveys such as the ACS and CPS, people are found
+to respond at different rates depending on demographics such as age, sex, and
+race or ethnicity.
+
+For each demographic subgroup in pseudopeople, we assumed the nonresponse rate
+in the Decennial Census was equal to the *net* rate of undercount (ignoring
+duplication) estimated in [Census_PES]_. Net undercount rates for age/sex were
+combined additively with rates for race/ethnicity, and demographic subgroups
```

Comment:
```suggestion
duplication) estimated in [Census_PES]_. Net undercount effects of age/sex were
combined additively with effects of race/ethnicity, and demographic subgroups
```

Is this too confusing? It's the difference, subgroup_undercount - overall_undercount, that gets calculated for each age/sex, and added to the same for race/ethnicity.

Reply:
Oh, thanks, I think this is better! I think it's more confusing to say "rates" since that's not exactly right.